<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>PopUp_Dialog</class>
 <widget class="QDialog" name="PopUp_Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>413</width>
    <height>105</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Alert</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0">
    <widget class="QFrame" name="frame">
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <widget class="QLabel" name="popup_label">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>30</y>
        <width>331</width>
        <height>17</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>12</pointsize>
       </font>
      </property>
      <property name="text">
       <string>TextLabel</string>
      </property>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
